src/backend/sideload/games.ts

```diff
@@ -240,42 +229,6 @@ export async function launchApp(appName: string): Promise<boolean> {
   return false
 }
 
-export async function moveInstall(
```

Comment:
I wonder if this is ok to be removed or if it means that at some point we lost the code to trigger the move of the installation for sideloaded apps?

cc @flavioislima to confirm

Reply:
Got fixed by merging with newest commits.

Reply:
Ok vscode github pull request extension tricked me. If i click on your comment it jumps to this function and shows me that it is added?

Reply:
@flavioislima do we need the moveInstall function for sideload games?